ministryofjustice/correspondence_tool_staff
# CT-2975 Show third party name in case list

## Description
CT-2975 Third party name in case list

## Self-review checklist


### Screenshots
State 1 - Non third party case displayed:

![1  requester-requested](https://user-images.githubusercontent.com/6988369/90143498-993c4f00-dd75-11ea-8358-9d954a88f4e7.png)

State 2 - Third party case with company name displayed:

![2  third-party-company-name](https://user-images.githubusercontent.com/6988369/90143630-c25cdf80-dd75-11ea-83bf-4ffe6f7b3606.png)

State 3 - Third party case with contact name (no company name entered) displayed

![3  third-party-contact-name](https://user-images.githubusercontent.com/6988369/90143799-f0daba80-dd75-11ea-9ed9-5b49398aa826.png)

### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&projectKey=CT&modal=detail&selectedIssue=CT-2975

### Deployment


### Manual testing instructions

State 1:
Create a case having "Requested on someone else's behalf?" set to 'No' and "where should it be sent?" set to either 'Data subject' or 'Third party'.  The cases list page will display the case requester's name duplicated in the "Request" column

State 2:
Create a case having "Requested on someone else's behalf?" set to 'Yes'.  Enter all required details, including a representative and company name.  The SAR can be sent to either data Subject or third party.  The cases list page will display the case requester's name with the third party company name below in the "Request" column

State3:
Edit the case created above byy deleting the company name.  The cases list page will now display the case requester's name with the third party contact name below in the "Request" column.
